apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -270,9 +271,7 @@ const BookingPage = ({
   }, []);
 
   const mutation = useMutation(createBooking, {
-    onSuccess: async (responseData) => {
-      const { uid } = responseData;
-
+    onSuccess: async (responseData: BookingResponse) => {
```

Comment:
[Prefer inferred types over explicit ones](https://handbook.cal.com/engineering/best-practices/prefer-inferred-types-over-explicit-ones)

```suggestion
    onSuccess: async (responseData) => {
```